copyright

```diff
@@ -1598,6 +1598,37 @@ Files:
 Copyright: Michael Zahniser <mzahniser@gmail.com>
 License: CC-BY-SA-4.0
 
+Files:
```

Comment:
You’ll have problems with the city files. It probably wouldn’t hurt to move it all close to the end of the file.

Reply:
Just moved them to the end, after any other files in the same naming scheme

Reply:
Great, the workflows should be working now when they get run.

Reply:
(you can resolve this by pressing the `Resolve Conversation` button)